storybook-testing/README.md

```diff
@@ -0,0 +1,33 @@
+#  Component Library Testing Strategy
```

Comment:
pls move into the main readme under the presentational layer explanation

Reply:
should be fixed